### DISCUSSION TITLE
I will skip 1.3.1 do to reboot issues

### DISCUSSION THREAD

**Discussion by K9-711** [eduardofilo/RG350_adam_image#152] [Q&A]
I'm still on 1.3 which works perfect for my RG350m. I red about boot issues on 1.3.1 when the device put several times in sleep mode. 

Will this be solved in an upcoming version?
And I wonder if I then have to flash the entire image or if it will be possible to update the way it is from 1.3 to 1.3.1.

**Comment by eduardofilo** [comment]
I do not know. It largely does not depend on me. The image is a software reunion. Basically OpenDingux beta, RetroArch, SimpleMenu and a few independent applications and emulators. I don't develop any of those pieces (well yeah, Py Backup). In the image there is also some configuration, especially RetroArch and SimpleMenu. With the configuration-related problems, I do have a chance. But those related to the code are not in my hand. The problem with the sleep mode that you mention does not occur to me that it may be related to configuration, so I suppose that if it is solved or improved in the future it will be through the update of some of the packages in the image.

Usually the upgrade method is by flashing the image again. But precisely between 1.3 and 1.3.1 due to the small amount of changes, there is exceptionally a direct update method. You have the instructions in the release change log:

https://github.com/eduardofilo/RG350_adam_image/releases/tag/v1.3.1
